.gitignore

```diff
@@ -158,3 +158,5 @@ cython_debug/
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
+
+examples/*.safetensors
```

Comment:
Should we use examples/**/*.safetensors?

If I’m reading correctly, this is really for the benefit of users who follow the examples, but it won’t ignore another other files in the model folder, so it might not be worth trying

Reply:
yep, good call! will do that